common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/TestShuffleDataContext.java

```diff
@@ -54,7 +54,7 @@ public void create() throws IOException {
       localDirs[i] = JavaUtils.createDirectory(root, "spark").getAbsolutePath();
 
       for (int p = 0; p < subDirsPerLocalDir; p ++) {
-        new File(localDirs[i], String.format("%02x", p)).mkdirs();
+        JavaUtils.createDirectory(new File(localDirs[i], String.format("%02x", p)));
```

Comment:
@dongjoon-hyun I moved the `createDirectory` to `JavaUtils` because of this place, since it's a test code, I can simply use `java.nio.Files.createDirectory` then we don't need to expose the method in `JavaUtils` publicly, please let me know which one you prefer.

Note: there are many places in Spark codebase directly call `java.nio.Files.createDirectory`